src/test/java/com/google/api/codegen/testdata/go_example_library.baseline

```diff
@@ -28,35 +28,53 @@ func ExampleNewClient() {
     ctx := context.Background()
     opts := []gax.ClientOption{ /* Optional client parameters. */ }
     c, err := library.NewClient(ctx, opts...)
-    _, _ = c, err // Handle error.
+    if err != nil {
+        // TODO: Handle error.
+    }
+    // TODO: Use client.
+    _ = c
 }
 
 func ExampleClient_CreateShelf() {
     ctx := context.Background()
     c, err := library.NewClient(ctx)
-    _ = err // Handle error.
+    if err != nil {
+        // TODO: Handle error.
+    }
 
     req := &google_example_library_v1.CreateShelfRequest{ /* Data... */ }
     var resp *google_example_library_v1.Shelf
     resp, err = c.CreateShelf(ctx, req)
```

Comment:
So the reason I originally added the `var resp` was because I wanted to hint the type of the response to the user in the example (admittedly, also because without the type, imports can occur that aren't necessary if the response type is in a different package than the request type, because of the way our import handler is designed).